src/data.cpp

```diff
@@ -625,7 +731,7 @@ std::vector <Interval> getTracked (
 
   // Assign an ID to each interval.
   for (unsigned int i = 0; i < intervals.size (); ++i)
-    intervals[i].id = intervals.size () - i;
+    intervals[i].id = intervals.size () - i + id_skip;
```

Comment:
Please use curly braces for `for` and `if` statements for new and modified code 😉